src/formatDate.js

```diff
@@ -50,7 +50,41 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArrFromFormat = date.split(fromFormat[fromFormat.length - 1]);
+  const dateArrToFormat = [];
+  const dateObject = {};
+
+  for (let i = 0; i < dateArrFromFormat.length; i++) {
+    dateObject[fromFormat[i]] = dateArrFromFormat[i];
+  }
+
+  for (let i = 0; i < toFormat.length - 1; i++) {
+    if (!(toFormat[i] in dateObject)) {
+      switch (toFormat[i]) {
+        case 'YYYY':
```

Comment:
Move `YYYY` and `YY` to variable please, it still not fixed from previous comments